src/Okta.Idx.Sdk.E2ETests/Helpers/TestUserHelper.cs

```diff
@@ -38,15 +40,59 @@ public async Task<TestUserProperties> GetActivePasswordUserAsync()
             };
         }
 
-        public TestUserProperties GetUnassignedUser()
+        public async Task<TestUserProperties> GetUnassignedUser()
         {
+            await CleanUpAsync();
+
+            var oktaUser = await _oktaHelper.CreateUnassignedUserIdentifiedWithPasswordAsync(_a18nProfile.EmailAddress, _configuration.UserPassword);
+            
             return new TestUserProperties()
             {
-                Email = _configuration.UnassignedUser,
-                Password = _configuration.UserPassword
+                Email = oktaUser.Profile.Email,
+                PhoneNumber = oktaUser.Profile.PrimaryPhone,
+                Password = _configuration.UserPassword,
             };
         }
 
+        public async Task<string> GetRecoveryCodeFromEmail()
+        {
+            for (int tries = 0; tries < 30; tries++) {
```

Comment:
did we choose 30 for any particular reason to retry?

Reply:
No, it's my guess that this will be enough. Usually mail comes in a couple of seconds.